src/Console/ConfigurationResolver.php

```diff
@@ -459,6 +459,45 @@ private function resolveFixers()
     {
         $this->fixers = $this->fixerFactory->useRuleSet($this->ruleSet)->getFixers();
 
+        // test if there are conflicting fixers configured
+        $conflictMap = $this->getFixersConflictMap();
```

Comment:
do you only want the conflict map  (and fetch method) in the Factory or the logic to check for conflicts as well?